developer-docs-site/docs/tutorials/full-node/update-fullnode-with-new-releases.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Update Fullnode With New Releases"
+slug: "update-fullnode-with-new-releases"
+sidebar_position: 11
+---
+
+# Update Fullnode With New Releases
```

Comment:
We should try to make this have separate instructions for Rust source users vs Docker users.

For Docker users, we should include a step where they run `docker pull` to get the latest version of the image.

Reply:
The Rust instructions I am pushing out to a later PR. Need to figure that out first.